sdk/search/search/src/serviceModels.ts

```diff
@@ -55,9 +55,44 @@ import {
   TagScoringFunction,
   TextWeights,
   ScoringFunctionAggregation,
-  RegexFlags
+  RegexFlags,
+  ConditionalSkill,
+  KeyPhraseExtractionSkill,
+  OcrSkill,
+  ImageAnalysisSkill,
+  LanguageDetectionSkill,
+  ShaperSkill,
+  MergeSkill,
+  EntityRecognitionSkill,
+  SentimentSkill,
+  SplitSkill,
+  TextTranslationSkill,
+  WebApiSkill,
+  DefaultCognitiveServicesAccount,
+  CognitiveServicesAccountKey
 } from "./generated/service/models";
 
+/**
+ * Options for a list skillsets operation.
+ */
+export interface ListSkillsetsOptions extends OperationOptions {
+  /**
+   * Selects which top-level properties of the skillsets to retrieve. Specified as a
+   * comma-separated list of JSON property names, or '*' for all properties. The default is all
+   * properties.
+   */
+  select?: string;
```

Comment:
I think we should make this (and the one below) a string array and then in the convenience layer join it with `,`

Reply:
Will do in the next commit.